File: leet code practice/Sum of Unique Elements.py
class Solution:
    def sumOfUnique(self, nums: List[int]) -> int:
        dic = defaultdict(list)
        for i in range(len(nums)):
            print(nums[i])
            if dic[nums[i]] :
                dic[nums[i]]+=1
            else:
                dic[nums[i]]=1
        sum = 0
        for key in dic :
            if dic[key]==1 :
                sum+=key
            
        return sum
                     
            
        

File: leet code practice/Sum of Unique Elements.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var sumOfUnique = function(nums) {
    var map={}
    for(let i=0;i<nums.length;i++){
        if(map[nums[i]]>0)
            map[nums[i]]++
        else{
            map[nums[i]]=1
        }
    }
    var sum=0
    for(key in map){
        if(map[key]===1)
            sum+=parseInt(key)
            
    }
    return sum;
};